README.md

```diff
@@ -111,7 +111,7 @@ const did = new DID({ resolver: KeyResolver.getResolver() })
 await did.resolve('did:key:...')
 ```
 
-### Creating a DID with attached CACAO
+### Using CACAO OCAPs to create a Key DID that can sign on behalf of a PKH DID.
```

Comment:
Title was good, just meant that we would add more context below!

Reply:
Oh woops okay

Reply:
done